.github/workflows/test.yml

```diff
@@ -39,10 +43,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9]
-        include:
-          - os: windows-latest
```

Comment:
We should add windows back for the unit/model tests, but it's probably ok to remove formatting checks on windows